2-1-2.py

```diff
@@ -0,0 +1,11 @@
+weight = float(input())
+growth = float(input())
```

Comment:
`height` would be a more appropriate name

Reply:
yes, you're right